compass/ocean/tests/soma/default/namelist.init

```diff
@@ -0,0 +1,2 @@
+config_vert_levels = 100
+config_vertical_grid = '100layerE3SMv1'
```

Comment:
```suggestion
config_vert_levels = 60
config_vertical_grid = '60layerPHC'
```
Maybe this is better?

Reply:
Maybe make this change in a follow-up PR?

Reply:
I think this should be
```suggestion
config_vert_levels = -1
config_vertical_grid = '60layerPHC'
config_soma_vert_levels = 60
```
I'll take a more careful look that the code to be sure.

Reply:
@xylar Just to note (for myself) but as we talked, config_vert_levels = 60 did not work, did not change the information in the namelist so need to set config_coma_vert_levels = 60 as you suggested. This worked.